packages/block-editor/src/components/rich-text/index.js

```diff
@@ -406,7 +408,11 @@ export function RichTextWrapper(
 				aria-readonly={ shouldDisableEditing }
 				{ ...props }
 				aria-label={
-					bindingsPlaceholder || props[ 'aria-label' ] || placeholder
```

Comment:
@gziolo, this change broke the `Cover` block tests.

But this situation is confusing for me: what is the priority here?
- Do we always want to respect the `aria-label`?
- Do we want to have the custom placeholder as the `aria-label`?
- Why would we want the bindings placeholder to take priority over the `aria-label`?

I'd love it if you could clarify this to me.

Reply:
That's probably this code:

<img width="1159" alt="Screenshot 2024-09-12 at 13 44 43" src="https://github.com/user-attachments/assets/806b9c03-83da-4319-ade3-58f13a78b9e8">

If we have a bound block, we rather need to show something else. However, it looks like `aria-label` should be the original one when the attribute isn't connected.

Reply:
Gotcha @gziolo. It seems weird to me that we want to display the bindings placeholder as `aria-label` considering a custom placeholder exists, but I reverted the `aria-label` changes and this PR should be good to go. Please take another look!